app-timer-run/build.gradle

```diff
@@ -19,4 +19,6 @@ dependencies {
     implementation libs.hilt.android
     kapt libs.hilt.compiler
 
+    implementation libs.glide
+    annotationProcessor libs.compiler
```

Comment:
The compiler is redundant here because we only use Glide to load images.

Reply:
My bad. I admit this one was a blind copy-pasting of some tutorial